app/routes/transaction.py

```diff
@@ -13,6 +19,28 @@ def transaction():
         else:
             transactions = Transaction.query.filter_by(merchant_id=current_user.id)
 
-        return render_template("transaction.html", transactions=transactions)
+        recentTransactions = []
+        for x in range(0, 4):
+            d = date.today() - timedelta(x)
+            recentTransactions.append(Transaction.query.filter_by(created=d).count())
+        transactionsByMerchant = {}
+        merchant01 = Merchant.query.filter_by(id=vente01_id).first()
```

Comment:
si vente_01id et le id du merchant
```suggestion
        merchant01 = Merchant.query.get(vente01_id)